src/modules/Pitcher/pitcher.py

```diff
@@ -1,51 +1,94 @@
 """Pitcher module"""
 
-import crepe
-from scipy.io import wavfile
-
-from modules.console_colors import (
-    ULTRASINGER_HEAD,
-    blue_highlighted,
-    red_highlighted,
-)
+import sys
+
+import torchcrepe
+from torch.cuda import OutOfMemoryError
+
+from modules.console_colors import ULTRASINGER_HEAD, blue_highlighted, red_highlighted
 from modules.Pitcher.pitched_data import PitchedData
 
+DEFAULT_MODEL: str = "full"
+SUPPORTED_MODELS: list[str] = ["tiny", "full"]
+TIMES_DECIMAL_PLACES: int = 3
 
-def get_pitch_with_crepe_file(filename: str, step_size: int, model_capacity: str) -> PitchedData:
+
+def get_pitch_with_crepe_file(
```

Comment:
do we need to dispose trouchcrepe? because the allocation is held till the program ends.